test/e2e/testdata/http-outputs.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Workflow
+metadata:
+  generateName: http-outputs-
+  labels:
```

Comment:
please delete all the labels, not needed

Reply:
deleted